src/sentry/integrations/jira/notify_action.py

```diff
@@ -71,26 +71,26 @@ def get_label_form(data):
         :return: String
         """
 
-        labels = ["Create a Jira ticket in the {jira_integration} account"]
-
-        if data.get("project"):
-            labels.append("and {project} project")
-        if data.get("issuetype"):
-            labels.append("of type {issuetype}")
-        if data.get("components"):
-            labels.append("with components {components}")
-        if data.get("duedate"):
-            labels.append("and due date {duedate}")
-        if data.get("fixVersions"):
-            labels.append("with fixVersions {fixVersions}")
-        if data.get("assignee"):
-            labels.append("assigned to {assignee}")
-        if data.get("reporter"):
-            labels.append("reported by {reporter}")
-        if data.get("labels"):
-            labels.append("with the labels {labels}")
-        if data.get("priority"):
-            labels.append("priority {priority}")
+        labels = ["Create a Jira issue in {jira_integration} "]
+
+        # if data.get("project"):
```

Comment:
I commented this all out just in case we still need it and I didn't know why, but if these are commented in it was creating a "part" for each one and we just want the Jira instance.